app/src/main/java/woowacourse/omok/NotationDao.kt

```diff
@@ -0,0 +1,88 @@
+package woowacourse.omok
+
+import android.content.Context
+import android.database.Cursor
+import android.database.sqlite.SQLiteDatabase
+import android.provider.BaseColumns
+import androidx.core.content.contentValuesOf
+import woowacourse.omok.FeedReaderContract.FeedNotation
+import woowacourse.omok.FeedReaderContract.SQL_DROP_TABLE
+import woowacourse.omok.domain.omok.model.Place
+
+class NotationDao(context: Context) {
+    private val dbHelper = FeedReaderDbHelper(context)
+
+    fun save(place: Place): Place {
```

Comment:
```suggestion
class NotationDao<T>(context: Context) {
    private val dbHelper = FeedReaderDbHelper(context)

    fun save(place: T): T {
```

이건 정말 제 과한 욕심이라서, 아 이럴수도 있구나 하고 넘어가주세요.
generic 을 활용하면 좀더 재사용가능한 Dao 를 만들 수 있을 것 같습니다.
out 과 in 을 활용하면 쿼리문을 만드는데 필요한 최소한의 메서드를 인터페이스로 지정할 수도 있고요.
그럴 수도 있겠다 하고 넘어가주시면 됩니다.